### PR TITLE
Adding quote exercise

### PR DESCRIPTION
**Pull request by tjbladez**
Changing second development exercise to the backbone based list.

### CODE REVIEW

**Comment by DrkCloudStrife** [comment]
No initial markup provided for quotes?

Maybe provide a link to the style-guide so they can also get a grip on that.

File: README.markdown [hunk 1]
```diff
@@ -5,26 +5,25 @@ If you have any questions feel free to email developers@benchprep.com
...
 
 Please, write code to simulate [jquery accordion](http://docs.jquery.com/UI/Accordion) plugin behavior with those sections.
 By default current setup comes with qunit, feel free to drop and use your own testing framework or write all your code as coffescript.
 
-### Nested Menu
+### Simple Backbone List
 
-In `menu.html` you will find `nestedMenu` structure that contains structure for nested menu that you need to build. Initially
-you will see `study`,`practice`,`test` headers. 
+In `quotes.html` you will need to build a simple backbone view to list given quotes.
```
Comment (nick-desteffen) [inline README.markdown]: In `quotes.html` you will need to build a simple **backbone.js** view to list given quotes.

File: javascript/simple_backbone/quotes.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+                    "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <script src="http://code.jquery.com/jquery-latest.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.1.2/backbone-min.js"></script>
```
Comment (nick-desteffen) [inline javascript/simple_backbone/quotes.html]: Add underscore

**Comment by tjbladez** [comment]
Nope, candidate needs to figure out based on the supplied json (imagine response from some other "service" they are consuming) As to the style-guide i think for the purpose of this exercise wont be to test their css, any sort of basic styling would be accepted. However, we should discuss and create front-end (landing page + css) exercise.

**Comment by DrkCloudStrife** [comment]
Sounds good.

**Comment by DrkCloudStrife** [comment]
Very good.

**Pull request merged**